nevergrad/benchmark/plotting.py

```diff
@@ -299,7 +302,7 @@ class XpPlotter:
     """
 
     def __init__(
-        self, optim_vals: Dict[str, Dict[str, np.ndarray]], title: str, name_style: Optional[Dict[str, Any]] = None, xaxis: str = "budget"
+        self, optim_vals: Dict[str, Dict[str, np.ndarray]], title: str, name_style: Optional[Dict[str, Any]]=None, xaxis: str="budget"
```

Comment:
nits: what triggered these corrections? When typing is present, there must be spaces around =

Reply:
Fixed, thx.